docs/src/main/sphinx/functions/conversion.rst

```diff
@@ -78,6 +78,13 @@ Unit    Description   Value
 ``YB``  Yottabytes    1024\ :sup:`8`
```

Comment:
I think it's wrong to use 1024 in `parse_data_size`. It should either use 1000, or units should be `KiB`, `MiB` etc.

See the references on this page: https://en.wikipedia.org/wiki/Kilobyte

The question is - should I fix it in this PR or use the same invalid base in `format_data_size`? Note that changing the base in `parse_data_size` is a breaking change.

Reply:
I added correct units in another commit.

Reply:
The same issue exists in https://github.com/airlift/units/blob/master/src/main/java/io/airlift/units/DataSize.java but since it's used for configuration it's a really risky change to make there. Related docs page: https://trino.io/docs/current/admin/properties.html#data-size

Reply:
I opened https://github.com/airlift/units/issues/26